docs/current_docs/quickstart/853930-directories.mdx

```diff
@@ -15,21 +15,25 @@ This opens powerful applications to Dagger Functions. For example, a function th
 
 ### Call a builder function
 
-Try calling this Go builder function:
+Try calling this Go builder Dagger Function, which builds the Dagger CLI:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project https://github.com/dagger/dagger --args ./cmd/dagger
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.10 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger --os=$(uname -s | awk '{print tolower($0)}')
```

Comment:
I'll leave it as is, because the fact that it produces this type of output provokes the discussion of how to make it more useful and therefore provides a bridge to the "function chaining" topic in the next section. But thank you very much for the thought anyway!